app/helpers/increment.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
```

Comment:
Please remove this use pre-existing helper. Check this https://github.com/DockYard/ember-composable-helpers#inc